Gemfile

```diff
@@ -38,8 +38,8 @@ group :system_tests do
   # This patch is required to fix Beaker's broken `aio` handling and provide support for SuSE
   # If you want to use 'bundle update --local', comment out this line and uncomment the next line
   # If you do this, please remember not to check in that change.
-  gem 'beaker', :git => 'https://github.com/trevor-vaughan/beaker.git', :branch => 'BKR-978-2.51.0'
-  # gem 'beaker'
+  gem 'google-api-client'
```

Comment:
@jeefberkey The only way to successfully test this is in GCE since you need a console to debug issues. This is part of the GCE support that was added in the test.